### PR TITLE
Issue #6207: Added XPath IT Regression Tests for InterfaceTypeParameterName

### PR DESCRIPTION
**Pull request by kkoutsilis**
Part of #6207 
Added XPath IT Regression Tests for [InterfaceTypeParameterName](https://checkstyle.org/checks/naming/interfacetypeparametername.html#InterfaceTypeParameterName)

### CODE REVIEW

**Comment by romani** [comment]
@kkoutsilis , please review https://app.circleci.com/pipelines/github/checkstyle/checkstyle/28161/workflows/81913fe6-9ce9-4523-b10d-996d969db516/jobs/700361/artifacts

try to fix.

**Comment by kkoutsilis** [comment]
> @kkoutsilis , please review https://app.circleci.com/pipelines/github/checkstyle/checkstyle/28161/workflows/81913fe6-9ce9-4523-b10d-996d969db516/jobs/700361/artifacts
> 
> try to fix.

Done

File: src/it/resources/org/checkstyle/suppressionxpathfilter/interfacetypeparametername/InputXpathInterfaceTypeParameterNameOnlySingleLetter.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.checkstyle.suppressionxpathfilter.interfacetypeparametername;
+
+public class InputXpathInterfaceTypeParameterNameOnlySingleLetter {
+  interface FirstInterface<T> {} // ok
+  interface SecondInterface<t> {} // ok
+  interface ThirdInterface<Type> {} // warn
```
Comment (romani) [inline src/it/resources/org/checkstyle/suppressionxpathfilter/interfacetypeparametername/InputXpathInterfaceTypeParameterNameOnlySingleLetter.java:6]: Please remove ok comments, it is just noise.
We need only warn to be instantly visible, as red flag.

Please remove from all inputs of your PR

**Review by romani — CHANGES_REQUESTED** [review]
Last:

**Review by romani — CHANGES_REQUESTED** [review]
After second look.

All xpath looks very similar to each other.
We are testing xpath, so ideally we should have different by structure xpath queries. 
Can we make second input as interface as top level public type?
Third as interface declaration in inner class or lambda.

**Review by romani — APPROVED** [review]
ok to merge if ci pass

**Comment by romani** [comment]
@kkoutsilis , please help with "good fourth issue"

**Comment by kkoutsilis** [comment]
> @kkoutsilis , please help with "good fourth issue"

Hey @romani, I missed that message and started on SuppressWarnings. Will raise that PR and move on after. Thank you :)

**Comment by romani** [comment]
@kkoutsilis ,
please send PR for what you started ! thanks a lot for your help. We do not limit contributors, but try to hint them that contributors can move forward with more complicated tasks if they fill firm with certain level and now need to get bored.

**Pull request merged**